license-maven-plugin/src/test/java/com/mycila/maven/plugin/license/LicenseSetTest.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2008 Mycila (mathieu.carbou@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mycila.maven.plugin.license;
+
+import org.apache.maven.plugin.testing.stubs.MavenProjectStub;
+import org.junit.Test;
+
+import java.io.File;
+
+public class LicenseSetTest {
+
+    @Test
+    public void multipleLicenseSets() throws Exception {
```

Comment:
I would definitely add more tests, especially to test when the user mixes some "legacy" or "base" config with 1 and/or 2 sets.